### DISCUSSION TITLE
finish build Insructions，what should i do next

### DISCUSSION THREAD

**Discussion by gongkaijie** [iTwin/itwinjs-core#280] [Q&A] [answered]
Hello, It's the first time For me to contact the IModel. Now I'm running the fifth step in build Insructions. May I ask how to run the code next

**Comment by calebmshafer** [comment]
@gongkaijie not sure I understand your question.  Are you asking how to look at an iModel from iModelHub? If so, have you seen our [Getting Started Guides](https://www.itwinjs.org/getting-started/online-quickstart/#2-get-a-sample-imodel).  They will walk you through the steps of setting up one of our samples and connecting to iModelHub to view your, or a sample, iModel.

**Reply by shenshengyi** [reply]
> Hello, It's the first time For me to contact the IModel. Now I'm running the fifth step in build Insructions. May I ask how to run the code next

Hello, if you are a Chinese user, you can visit our Bentley Chinese Forum and ask questions about imodel, and someone will answer them for you.

Links to imodeljs CHINAFIRST forum are as follows,Here you can ask various development questions about imodel:
https://communities.bentley.com/communities/other_communities/chinafirst/f/itwin-bridge-imodel-js-itwin-service#pifragment-25903=1
The links for getting started with imodeljs programming are as follows，Here you can read about imodel JS programming:
https://communities.bentley.com/communities/other_communities/bdn_other_communities/w/chinabdn-wiki/47328/imodel-js

**Comment by shenshengyi** [comment, marked as answer]
> Hello, It's the first time For me to contact the IModel. Now I'm running the fifth step in build Insructions. May I ask how to run the code next

If you want to run the instance code, please check this tutorial.

Links are as follows:
https://communities.bentley.com/communities/other_communities/bdn_other_communities/w/chinabdn-wiki/50995/imodel-js
